autologout.module

```diff
@@ -160,8 +168,9 @@ function autologout_autologout_prevent() {
  * Implements hook_autologout_refresh_only().
  */
 function autologout_autologout_refresh_only() {
-  // Check to see if an open admin page will keep login alive.
-  if (!\Drupal::config('autologout.settings')->get('autologout_enforce_admin') && path_is_admin(current_path())) {
+  $url = Url::fromRoute('<current>');
+  $route = explode('/',$url->toString());
+  if (!Drupal::config('autologout.settings')->get('autologout_enforce_admin') && \Drupal::service('router.admin_context')->isAdminRoute(\Drupal::routeMatch()->getRouteObject())) {
```

Comment:
Can you get the values in a variable first and then make the comparison?
Would make it more readable.